src/elements/table/table-wrapper/table-wrapper.scss

```diff
@@ -0,0 +1,36 @@
+@use '../../core/styles/index' as sbb;
```

Comment:
Apparently, scrollbar styling is not supported anymore since iOS 14. 
The sad thing is that `@supports` still returns positive so we don't have a way to discern which OS we are on :|
I think the time has come where we ask to drop the border-radius on the table 😞

Reply:
We have the same issue on Firefox :/